packages/eslint-config-nimble/components.js

```diff
@@ -27,6 +27,11 @@ export const componentsNimbleConfig = defineConfig([
                     paths: [...restrictedFastImports()],
                 },
             ],
+
+            // Strict rules components have not adopted yet
```

Comment:
Am I reading this correctly that these are things that the NI styleguide recently started enforcing but we're disabling them temporarily due to time constraints / scoping? If so, could you create and link to a tech debt issue to track fixing the violations and removing this config.